pina/equation/__init__.py

```diff
@@ -0,0 +1,12 @@
+__all__ = [
+    'SystemEquation',
+    'Equation',
+    'FixedValue',
+    'FixedGradient',
+    'FixedFlux',
+    'Laplace',
+]
+
+from .equation import *
+from .equation_factory import *
+from .system_equation import *
```

Comment:
Never use "import *"!

Reply:
solved in [e6acebd](https://github.com/mathLab/PINA/pull/141/commits/e6acebd6da17a1cc8ee4173942efbc128ea78200)